mempool/src/mempool_interface_impl/pool.rs

```diff
@@ -22,7 +22,6 @@ use std::time::Duration;
 use common::chain::Block;
```

Comment:
Please rename this file to mod.rs and put it in the pool directory

Reply:
Done.